Enable preview mode hug_parens_with_braces_and_square_brackets.

From: Charles <peacech@gmail.com>


---
 src/black/linegen.py |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/black/linegen.py b/src/black/linegen.py
index 46945ca..2ba838c 100644
--- a/src/black/linegen.py
+++ b/src/black/linegen.py
@@ -864,7 +864,7 @@ def _first_right_hand_split(
 
     body: Optional[Line] = None
     if (
-        Preview.hug_parens_with_braces_and_square_brackets in line.mode
+        True # Preview.hug_parens_with_braces_and_square_brackets in line.mode
         and tail_leaves[0].value
         and tail_leaves[0].opening_bracket is head_leaves[-1]
     ):
